apps/publish_mdm/forms.py

```diff
@@ -431,3 +448,70 @@ class Meta:
     Organization, TemplateVariable, form=TemplateVariableForm, extra=0
 )
 TemplateVariableFormSet.deletion_widget = CheckboxInput
+
+
+class CentralServerForm(forms.ModelForm):
+    """A form for adding or editing a CentralServer."""
+
+    class Meta:
+        model = CentralServer
+        fields = "__all__"
+        widgets = {
+            "password": forms.widgets.PasswordInput,
```

Comment:
Perhaps we should require re-entering the `email` as well.

If allowing save (e.g., edit to URL only) without re-entering these values is possible, it would be a nice-to-have.